scripts/templates/variables.tf.j2

```diff
@@ -53,7 +53,7 @@ variable "{{ id }}_runbook_url" {
 variable "{{ id }}_disabled" {
   description = "Disable all alerting rules for {{ id }} detector"
   type        = bool
-  default     = null
+  default     = {% if disabled is defined %}{{ disabled }}{% else %}null{% endif %}
```

Comment:
I don't think it is explicit/safe enough.
the values.yaml file is provided to be copyable and works without modification.

as you define this in values.yaml so it will disable all alerting by default and this is not the desirable behavior I think.

```suggestion
  default     = {% if disabled is boolean and disabled is sameas true %}true{% else %}null{% endif %}
```

Reply:
it seems we do not need the first test so this is enough: 

```suggestion
  default     = {% if disabled is sameas true %}true{% else %}null{% endif %}
```